File: lqzxc-iOS/lqzxc/lqzxc/BikeStie.swift
//
//  BikeStie.swift
//  lqzxc
//
//  Created by jyb on 15/7/25.
//  Copyright (c) 2015年 jyb. All rights reserved.
//

import UIKit

class BikeStie: NSObject {
    var id: Int // 顺序号(外部编号)
    var mid: String// 网点编号(内部编号)
    var gpsx: String// 经度
    var gpsy: String// 纬度
    var pt: CLLocationCoordinate2D// 经纬度标识
    var netName: String// 网点名
    var netType: String// 网点类型
    var netStatus: String// 网点状态
    var netLevel: String// 网点等级
    var address: String// 地址
    var trafficInfo: String// 周边公交信息
    var bicycleCapacity: Int// 总数
    var bicycleNum: Int// 库存
    var imageAttach: String// 图片相对路径
    var visited: Int// 访问数
    var time: String// 访问时间
    
    override init() {
        id = 0
        mid = ""
        gpsx = ""
        gpsy = ""
        pt = CLLocationCoordinate2D()
        netName = ""
        netType = ""
        netStatus = ""
        netLevel = ""
        address = ""
        trafficInfo = ""
        bicycleCapacity = 0
        bicycleNum = 0
        imageAttach = ""
        visited = 0
        time = ""
    }
}


class ListModel: NSObject {
    var count: Int
    var page: Int
    var bikeSties: NSMutableArray
    
    override init() {
        count = 0
        page = 0
        bikeSties = NSMutableArray()
    }
}

File: lqzxc-Android/lqzxc/app/src/main/java/com/lqzxc/modal/BikeStie.java
package com.lqzxc.modal;

import com.baidu.platform.comapi.basestruct.GeoPoint;
/**
 * 自定义站点数据结构
 * @author QQ472950043
 **/
public class BikeStie{
	int id;// 顺序号(外部编号)
	String mid;// 网点编号(内部编号)
	String gpsx;// 经度
	String gpsy;// 纬度
	GeoPoint mGeoPoint;// 经纬度标识
	String netName;// 网点名
	String netType;// 网点类型
	String netStatus;// 网点状态
	String netLevel;// 网点等级
	String address;// 地址
	String trafficInfo;// 周边公交信息
	int bicycleCapacity;// 总数
	int bicycleNum;// 库存
	String imageAttach;// 图片相对路径
	int visited;// 访问数
	String time;// 访问时间

	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getMid() {
		return mid;
	}
	public void setMid(String mid) {
		this.mid = mid;
	}
	/**
	 * 经度
	 */
	public String getGpsx() {
		return gpsx;
	}
	/**
	 * 经度
	 */
	public void setGpsx(String gpsx) {
		this.gpsx = gpsx;
	}
	/**
	 * 纬度
	 */
	public String getGpsy() {
		return gpsy;
	}
	/**
	 * 纬度
	 */
	public void setGpsy(String gpsy) {
		this.gpsy = gpsy;
	}
	public GeoPoint getmGeoPoint() {
		return mGeoPoint;
	}
	public void setmGeoPoint(GeoPoint mGeoPoint) {
		this.mGeoPoint = mGeoPoint;
	}
	public String getNetName() {
		return netName;
	}
	public void setNetName(String netName) {
		this.netName = netName;
	}
	public String getNetType() {
		return netType;
	}
	public void setNetType(String netType) {
		this.netType = netType;
	}
	public String getNetStatus() {
		return netStatus;
	}
	public void setNetStatus(String netStatus) {
		this.netStatus = netStatus;
	}
	public String getNetLevel() {
		return netLevel;
	}
	public void setNetLevel(String netLevel) {
		this.netLevel = netLevel;
	}
	public String getAddress() {
		return address;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	public String getTrafficInfo() {
		return trafficInfo;
	}
	public void setTrafficInfo(String trafficInfo) {
		this.trafficInfo = trafficInfo;
	}
	public int getBicycleCapacity() {
		return bicycleCapacity;
	}
	public void setBicycleCapacity(int bicycleCapacity) {
		this.bicycleCapacity = bicycleCapacity;
	}
	public int getBicycleNum() {
		return bicycleNum;
	}
	public void setBicycleNum(int bicycleNum) {
		this.bicycleNum = bicycleNum;
	}
	public String getImageAttach() {
		return imageAttach;
	}
	public void setImageAttach(String imageAttach) {
		this.imageAttach = imageAttach;
	}
	public int getVisited() {
		return visited;
	}
	public void setVisited(int visited) {
		this.visited = visited;
	}
	public String getTime() {
		return time;
	}
	public void setTime(String time) {
		this.time = time;
	}
}
